src/into.js

```diff
@@ -1,8 +1,8 @@
 const curry = require('./curry')
 const transduce = require('./transduce')
-const getAddReducer = require('./internal/getAddReducer')
+const getAddReducer = require('../internal/getAddReducer')
```

Comment:
Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you